File: frameworks/node/hono/Implementation/routes/json.ts
import type { Routes } from "../app.ts";

/** json: a payload the framework already holds, written by c.json with JSON.stringify. */
const json: Routes = (app, p) => {
  app.get("/json/small", (c) => c.json(p.small));

  app.get("/json/medium", (c) => c.json(p.medium));

  app.get("/json/large", (c) => c.json(p.large));
};

export default json;

File: frameworks/python/django-rest-framework/Implementation/views/json.py
from django.conf import settings
from rest_framework.decorators import api_view
from rest_framework.response import Response

# json: a payload the framework already holds, rendered by DRF's JSONRenderer on every request.
# Three static routes rather than one with a capture, so the router pays no capture here.
P = settings.PAYLOADS


# rb:handler json.small,cors.scoped
@api_view(["GET"])
def small(request):
    return Response(P.small)


# rb:handler json.medium
@api_view(["GET"])
def medium(request):
    return Response(P.medium)


# rb:handler json.large
@api_view(["GET"])
def large(request):
    return Response(P.large)
